Python/Sorting_Algorithms/bubble_sort.py

```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort 
```

Comment:
- Please delete algorithms except for Bubble, selection, insertion. 
- Update the readme of the Python directory with your entries for Bubble, selection, insertion

Reply:
Yeah I made the required changes and committed the changes. Hope you are okay with them.
Thank you.